api/lambdas/withings/index.js

```diff
@@ -52,9 +52,11 @@ const lookup = async address => {
 };
 
 async function forwardCallToServer(req) {
-  console.log(`Verified! Calling server...`);
+  const convertParams = new URLSearchParams(req.body);
```

Comment:
I'm not sure the approach will work - trying it out in a ts playground I get this:

```
const convertParams = new URLSearchParams("userid=32542297&startdate=1685475739&enddate=1685475740&appli=1");
console.log(convertParams); // [LOG]: URLSearchParams: {}
```

[https://www.typescriptlang.org/play?ts=4.9.5#code/PTAEHUFMBsGMHsC2lQBd5oBYoCoE8AHSAZVgCcBLA1UAB[…]AbIK5pHhQE58zphlh9CgEIhocJLmWJAXoiIDBxsQsmKwODwgkA](https://www.typescriptlang.org/play?ts=4.9.5#code/PTAEHUFMBsGMHsC2lQBd5oBYoCoE8AHSAZVgCcBLA1UABWgEM8BzM+AVwDsATAGiwoBnUENANQAd0gAjQRVSQAUCEmYKsTKGYUAbpGF4OY0BoadYKdJMoL+gzAzIoz3UNEiPOofEVKVqAHSKymAAmkYI7NCuqGqcANag8ABmIjQUXrFOKBJMggBcISGgoAC0oACCbvCwDKgU8JkY7p7ehCTkVDQS2E6gnPCxGcwmZqDSTgzxxWWVoASMFmgYkAAeRJTInN3ymj4d-jSCeNsMq-wuoPaOltigAKoASgAywhK7SbGQZIIz5VWCFzSeCrZagNYbChbHaxUDcCjJZLfSDbExIAgUdxkUBIursJzCFJtXydajBRQITiCGiUvRkVC0RwMRDCAC8-UgEgeL2IHjIGkZZGZggAFAAidiCb4UbisgDMACYAKwAFgVCoAnAB2ABk1McqG4dUgrIAjAA2AAcqq1Sq1co1OpR3CNCjNVptdpVAAYdQwCAsKGaxQBKADcwUpgng7gC0HgzBFtO+DKZLPDwRUOGafK8iHgfQYwPYNC+bjMzHYDGYkH4sGg6kSRfgehEXjFAFFVsyFvoxUlsWLwA5UAByYQAOU5YqCKgA8l8yO8pfwa0dUAbIK5pHhQE58zphlh9CgEIhocJLmWJAXoiIDBxsQsmKwODwgkA)

Reply:
.

Reply:
Yeah, it seems to be missing `.toString()` when we pass it as a body... tried JSON.stringify and it didn't work either

Reply:
So I'm still not sure why, but @Orta21 confirmed this works.

Thought it was perhaps because the lambda is JS and I was using a TS env, but tried it in JS to no avail?